h5p-bildetema-words-grid-view/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "vite build --mode=development --watch --minify=false",
     "build:dev": "vite build --mode=development",
-    "build": "tsc && vite build",
+    "build": "vite build && tsc",
```

Comment:
Turning these around to make sure that we build+generate all type files before typechecking is performed.